README.rst

```diff
@@ -59,6 +60,10 @@ To update, run::
 
     $ pip install -U crash
 
+If you use Python 2.7 or 3.3 with a recent version of pip_, it will install
+only version ``0.23.x`` by default, because newer versions of this package are
+not compatible with Python 2.7/3.3 any more.
+
```

Comment:
Further below mentions that the `crash_standalone_latest` works with py2.7. Did we ensure we're keeping the standalone at `0.23`? If not, can we freeze one and link it as well as py27-crash-standalone or something like that?

Reply:
https://cdn.crate.io/downloads/releases/ keeps all versions of crash standalone.
I will update the README accordingly.

Reply:
@mfussenegger updated the readme with instructions what version to install for legacy python versions.